cobalt/updater/updater.cc

```diff
@@ -38,25 +38,15 @@
 namespace {
 
 void TaskSchedulerStart() {
-  base::TaskScheduler::Create("Updater");
+  base::ThreadPoolInstance::Create("Updater");
```

Comment:
This should be good because it's the same change as made in upstream for https://issues.chromium.org/issues/40622485 and https://issues.chromium.org/issues/40629609